x/concentrated-liquidity/swapstrategy/swap_strategy_test.go

```diff
@@ -145,15 +150,17 @@ func (suite *StrategyTestSuite) TestComputeSwapState_Inverse() {
 			zeroForOne:       false,
 			spreadFactor:     sdk.ZeroDec(),
 
-			// from token_in:   sqrt_next = sqrt_cur + token_in / liq2 = 70.72451318306962507883763621
-			expectedSqrtPriceNextOutGivenIn: sdk.MustNewDecFromStr("70.724513183069625078"), // approx 5001.96
-			// from token_out:  sqrt_next = liq2 * sqrt_cur / (liq2 - token_out * sqrt_cur)
-			expectedSqrtPriceNextInGivenOut: sdk.MustNewDecFromStr("70.724513183069625078"), // approx 5001.96
+			// get_next_sqrt_price_from_amount1_in_round_down(liquidity, sqrtPriceCurrent, tokenIn)
+			expectedSqrtPriceNextOutGivenIn: osmomath.MustNewDecFromStr("70.724513183069625078753200000000838853"), // approx 5001.96
+
+			// tokenOut = round_sdk_prec_down(calc_amount_zero_delta(liquidity, Decimal('70.724513183069625078753200000000838853'), sqrtPriceCurrent, False))
+			// get_next_sqrt_price_from_amount0_out_round_up(liquidity, sqrtPriceCurrent, tokenOut)
+			expectedSqrtPriceNextInGivenOut: osmomath.MustNewDecFromStr("70.724513183069625078753199315615320286"), // approx 5001.96
```

Comment:
I think this is simply due to the lack of precision. We used to not have enough precision to represent the difference. Now, we do.

It is fair to expect them to be slightly off due to math and rounding behavior being different between out-given-in and in-given-out swaps. However, we know that the epsilon between `expectedSqrtPriceNextOutGivenIn` and `expectedSqrtPriceNextInGivenOut` must be extremely small because `expectedSqrtPriceNextInGivenOut` is computed from `tokenOut` which is computed from  `expectedSqrtPriceNextOutGivenIn`.